lib/aptible/cli/subcommands/environment.rb

```diff
@@ -41,6 +41,27 @@ def self.included(thor)
                 end
               end
             end
+
+            desc 'environment:rename OLD_HANDLE NEW_HANDLE',
+                 'Rename an environment handle. In order for the new'\
+                 ' environment handle to appear in log drain/metric'\
+                 ' destinations, you must restart the apps/databases in'\
+                 ' this environment.'
+            define_method 'environment:rename' do |old_handle, new_handle|
+              env = ensure_environment(options.merge(environment: old_handle))
+              env.update!(handle: new_handle)
+              m1 = "In order for the new environment handle (#{new_handle})"\
+                   ' to appear in log drain and metric drain destinations,'\
+                   ' you must restart the apps and databases in this'\
+                   ' environment. Also be aware of the following resources'\
+                   ' that may need names adjusted:'
```

Comment:
How did you decide on this wording? I'm asking because, since you already warned about the drains above, re-warning about the metadata makes it almost sound like some additional updates need to be done, which feels a bit confusing, because there shouldn't be additional changes.

Reply:
I included it in the ticket and sought approval at the time (from a copy/paste from the initial thread). Definitely needed word-smithing as it's confusing. 

If it makes it easier I can drop those two bullets as it's mentioned in the preceding sentences.